### DISCUSSION TITLE
Loading Animation & Percentage for tracking process

### DISCUSSION THREAD

**Discussion by nijiinhell** [projectdiscovery/nuclei#3728] [Ideas]
I work with a lot of URLS with --list command or -u sometimes whatever. It'd be very good if there'd be a Loading Animation & Percentage for tracking process

**Comment by MetzinAround** [comment]
Cool Idea @nijiinhell. I moved this to discussions since this is likely a ways away from implementation and would require lots of discussion work from the OSS team. 

@ehsandeep tagging you just to notify you of this idea. Not sure if it's been worked on or discussed before.

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
